### PR TITLE
Add new library for algebraic parser and method to parse it

### PR DESCRIPTION
**Pull request by khansaad**

### CODE REVIEW

File: src/main/java/com/autotune/analyzer/k8sObjects/ObjFunctionParser.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        result = new Expression(objFuncData)
+                .with("request_sum", objFuncMap.get("request_sum"))
+                .and("request_count", objFuncMap.get("request_sum"))
+                .setPrecision(128)
+                .setRoundingMode(RoundingMode.UP)
+                .eval();
+        return result.toString();
+    }
+
+    private static String validate(String objFuncData, String valueType, Map<String, String> objFuncMap) {
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/k8sObjects/ObjFunctionParser.java:33]: The validate function should be called from the `ValidateAutotuneObject` as part of `sloInfo.getObjectiveFunction()` validation.

**Review by dinogun — COMMENTED** [review]
@khansaad Thanks for the PR, a few comments as you work through this

**Comment by dinogun** [comment]
Also you should create branches on your fork and not directly on the `kruize/autotune` repo. I would suggest that you open a new PR from your fork

**Comment by dinogun** [comment]
I dont see GPG signing of your commits, please make sure to use `git commit -S` after you have setup GPG keys

**Comment by khansaad** [comment]
Thanks for the feedback dino, will follow the same. 
I'm closing this and will create a new one after forking as discussed.

**Pull request closed without merging**